tests/jenkins/jobs/uploadMinSnapshotsToS3_Jenkinsfile.txt

```diff
@@ -13,33 +13,21 @@
                   uploadMinSnapshotsToS3.readYaml({file=tests/data/tar/builds/opensearch/manifest.yml})
                   BuildManifest.asBoolean()
                   uploadMinSnapshotsToS3.echo(Create .sha512 for Min Snapshots Artifacts)
-                  createSha512Checksums.sh({script=find tests/data/tar/builds/opensearch/dist -type f, returnStdout=true})
-                  createSha512Checksums.echo(Creating sha for opensearch-min-1.3.0-linux-x64.tar.gz)
-                  createSha512Checksums.sh({script=sha512sum opensearch-min-1.3.0-linux-x64.tar.gz, returnStdout=true})
-                  createSha512Checksums.sh({script=basename ccc, returnStdout=true})
-                  createSha512Checksums.writeFile({file=opensearch-min-1.3.0-linux-x64.tar.gz.sha512, text=bbb  
-bbb
-ccc
-})
-                  createSha512Checksums.echo(Creating sha for opensearch-dashboards-min-1.3.0-linux-x64.tar.gz)
-                  createSha512Checksums.sh({script=sha512sum opensearch-dashboards-min-1.3.0-linux-x64.tar.gz, returnStdout=true})
-                  createSha512Checksums.sh({script=basename ccc, returnStdout=true})
-                  createSha512Checksums.writeFile({file=opensearch-dashboards-min-1.3.0-linux-x64.tar.gz.sha512, text=bbb  
-bbb
-ccc
-})
-                  uploadMinSnapshotsToS3.echo(Start copying files)
+                  createSha512Checksums.sh({script=find tests/data/null/builds/opensearch/dist -type f, returnStdout=true})
+                  createSha512Checksums.echo(Not generating sha for bbb in tests/data/null/builds/opensearch/dist, doesn't match allowed types [.tar.gz, .zip, .rpm, .deb])
+                  createSha512Checksums.echo(Not generating sha for ccc in tests/data/null/builds/opensearch/dist, doesn't match allowed types [.tar.gz, .zip, .rpm, .deb])
+                  uploadMinSnapshotsToS3.echo(Start copying files: ver-1.2.2-SNAPSHOT arch-x64 plat-linux build-29 dist-null ext-tar.gz)
                   uploadMinSnapshotsToS3.sh(
-        cp -v tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64.tar.gz tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz
-        cp -v tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64.tar.gz.sha512 tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz.sha512
-        cp -v tests/data/tar/builds/opensearch/dist/../manifest.yml tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz.build-manifest.yml
-        sed -i "s/.tar.gz/-latest.tar.gz/g" tests/data/tar/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz.sha512
+        cp -v tests/data/null/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64.tar.gz tests/data/null/builds/opensearch/dist/opensearch-min-1.2.2-SNAPSHOT-linux-x64-latest.tar.gz
```

Comment:
Seems like this distribution type now become null since it comes from the manifest. Maybe you could add it in the test?

Reply:
I think this exactly show why it defaults to tar.gz.

Because somehow the distribution value is not coming out of build manifest correctly.

Will investigate.